### PR TITLE
Add data view

### PR DESCRIPTION
**Pull request by CohenIdo**

### CODE REVIEW

File: packages/k8s_cis/kibana/index_pattern/9129a080-7f48-11ec-8249-431333f83c5f.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "attributes": {
+    "title": "logs-k8s_cis.result*"
+  },
+  "coreMigrationVersion": "8.1.0",
+  "id": "9129a080-7f48-11ec-8249-431333f83c5f",
```
[the remark this answers is not in the document]
Comment (kfirpeled) [reply packages/k8s_cis/kibana/index_pattern/9129a080-7f48-11ec-8249-431333f83c5f.json:6]: and the file name is the same as the id (so once you add the prefix add it also to the file name)

File: packages/k8s_cis/kibana/index_pattern/9129a080-7f48-11ec-8249-431333f83c5f.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  "migrationVersion": {
+    "index-pattern": "8.0.0"
+  },
+  "type": "index-pattern",
+  "updated_at": "2022-01-27T08:10:19.277Z",
+  "version": "WzMwNDY5LDFd"
```
Comment (kfirpeled) [inline packages/k8s_cis/kibana/index_pattern/9129a080-7f48-11ec-8249-431333f83c5f.json:12]: what did generate the version number? you used a tool for that?

**Review by kfirpeled — COMMENTED** [review]
it makes me wonder, why don't we see `index_pattern` in any of the `integrations`

**Comment by CohenIdo** [comment]
I solve the [id comment](https://github.com/build-security/integrations/pull/5#discussion_r793444098). 
It is outdated because the filename was changed.

**Pull request merged**